accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/ui/AcToolUiService.java

```diff
@@ -44,11 +61,15 @@ public class AcToolUiService {
 
     public static final String PAGE_NAME = "actool";
 
-    static final String PATH_SEGMENT_DUMP = "dump.yaml";
+    static final String SUFFIX_DUMP_YAML = "dump.yaml";
+    static final String SUFFIX_USERS_CSV = "users.csv";
 
     @Reference(policyOption = ReferencePolicyOption.GREEDY)
     private ConfigDumpService dumpService;
 
+    @Reference(policyOption = ReferencePolicyOption.GREEDY)
+    private UserProcessor userExporter;
```

Comment:
What about calling the interface also`UserExpoerter`? Also would call the package of that service `userexporter`

Reply:
This service does not export (in the sense of serializing or converting) something. The actual export is done within this service.

Reply:
Just rename variable to userProcessor.